com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/utils/FileSystemUtils.java

```diff
@@ -202,7 +202,7 @@ public Path createFolders(Path path) {
     try {
       return Files.createDirectories(path);
     } catch (IOException e) {
-      log.error(e.getMessage());
+      log.error("Error creating folders for "+ path + ": " + e);
```

Comment:
In this case, it won't print the stack trace of the exception. To do that, you may use the method `Logger#error(String, Throwable)`